### PR TITLE
BE#23-hub-data-endpoint

### PR DESCRIPTION
**Pull request by SoySauceLabs**
# Words For The Wise:
- [ ] This endpoint has been tested _LIVE_ (there's no words there!) by Ms. Brimm, but it you're anxious to do more poking and prodding the endpoint can be tested with **Postman** using the JSON structure documented in **Project Documentation > Wiki+Reference > BE > Batch Data JSON Structure**.
### Otherwise...
- [ ] Check it.
- [ ] Ask Matt wth is going on.
- [ ] Ship it.

### CODE REVIEW

File: src/cassy.py [hunk 1]
```diff
@@ -62,6 +62,32 @@ def get_env_data(node_id, env_variable):
...
+    """
+    Inserts data from hub into database.
+    """
+
+    query = 'BEGIN BATCH\n'
+
+    try:
+        for data_point in env_data['hub_data']:
+            query = query + (
+                    'INSERT INTO ' + os.environ.get('DB_ENV_TABLE') +
```
Comment (kbrimm) [inline src/cassy.py]: Should we think about parameterizing the inputs here?

Comment (SoySauceLabs) [inline src/cassy.py]: Ahh yes, I can take care of that!

**Comment by SoySauceLabs** [comment]
BRRRRIIIINNNNGGG, BRRRRRRRRIIIINNNGGGG!!! We've now got a nice batch of...well, of prepared batch statements!

**Comment by kbrimm** [comment]
LOOKS GOOD YAY.

**Pull request merged**